builder/package.json

```diff
@@ -70,6 +70,7 @@
     "@types/node": "^14.6.1",
     "@types/supports-color": "^5.3.0",
     "rimraf": "~3.0.0",
+    "source-map-loader": "~1.0.2",
```

Comment:
You need to move this in the `dependencies` section because devDeps of devDeps are not pulled. So this package is a bit exotic in that regard. This is the reason for having `style-loader` or `worker-loader` for example in the dependencies.